myning/upgrades.yaml

```diff
@@ -106,6 +106,21 @@ armory_hints:
   costs:
     - 2000
 
+sort_by_value:
```

Comment:
could you help me understand the value of sorting by value? so you can sell cheap stuff first and not expensive stuff? why would someone want this?

Also the description says "get the option" but it doesn't look like there is an option, once you've bought the upgrade it just stays sorted

Reply:
Good point. I'll instead add it to settings. 

Use case is for people who want to only buy the top tier gear from the store, garden, or blacksmith.

Reply:
ahh that makes sense. It wasn't quite clear at first. How about tweaking the descriptions to something along the lines of "get the option to sort the store by best items first", "best equipment first", "best seeds first"

Reply:
They're technically at the bottom though. I feel like sorting by value makes sense. Is there a better word for "value"? "Quality?"

Reply:
if the why behind this upgrade is to make it easier to get the highest value items, then I feel like we could reverse the sort and just put them at the top? sorting by value descending. makes little difference that it's at the bottom, just don't have to press up to get to the bottom of the list. That way the upgrade sounds nicer as well

Reply:
The problem is that until TUI is done, the top of the door is often almost out of view or fully out of view. The bottom is always visible 😅

Reply:
that's a good point. We can leave it as-is then, and when tui is done change the description and reverse the sort if it still makes sense